### PR TITLE
Conviva:

### PR DESCRIPTION
**Pull request by numismatika**
- add livepassdl, livepassdl2, ^ and www subdomains

### CODE REVIEW

File: src/chrome/content/rules/Conviva.xml [hunk 1]
```diff
@@ -3,26 +3,20 @@
 
 		- (www.)
```
Comment (J0WI) [inline src/chrome/content/rules/Conviva.xml]: You should update this comment.
`(www.)` seems working now but there are still some broken ones.

Mismatch on:
static.conviva.com
lp.conviva.com

File: src/chrome/content/rules/Conviva.xml [hunk 2]
```diff
@@ -3,26 +3,20 @@
...
-
-
-	<rule from="^http://livepass(?:dl)?\.conviva\.com/"
-		to="https://livepass.conviva.com/" />
+	<target host="conviva.com" />
+	<target host="livepass.conviva.com" />
+	<target host="livepassdl.conviva.com" />
+	<target host="livepassdl2.conviva.com" />
+	<target host="pulse.conviva.com" />
+	<target host="www.conviva.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/Conviva.xml]: `community.` and `developer.` are working too

File: src/chrome/content/rules/Conviva.xml [hunk 3]
```diff
@@ -16,13 +10,15 @@
...
-
-	<rule from="^http://livepass(?:dl)?\.conviva\.com/"
-		to="https://livepass.conviva.com/" />
+	<target host="conviva.com" />
+	<target host="community.conviva.com" />
+	<target host="developer.conviva.com" />
+	<target host="livepass.conviva.com" />
+	<target host="livepassdl.conviva.com" />
+	<target host="livepassdl2.conviva.com" />
+	<target host="pulse.conviva.com" />
```
Comment (mnordhoff) [inline src/chrome/content/rules/Conviva.xml]: https://pulse-beta.conviva.com/ also seems to work

File: src/chrome/content/rules/Conviva.xml [hunk 4]
```diff
@@ -16,13 +10,16 @@
...
-	<rule from="^http://pulse\.conviva\.com/"
-		to="https://pulse.conviva.com/" />
+	<target host="conviva.com" />
+	<target host="community.conviva.com" />
+	<target host="developer.conviva.com" />
+	<target host="livepass.conviva.com" />
+	<target host="livepassdl.conviva.com" />
+	<target host="livepassdl2.conviva.com" />
+	<target host="pulse.conviva.com" />
+	<target host="pulse-beta.conviva.com/" />
```
Comment (J0WI) [inline src/chrome/content/rules/Conviva.xml]: You need to remove the trailing slash on the host.

**Comment by numismatika** [comment]
@J0WI  should be better now after the rebase, although the test timed out on my branch.

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**